change/@itwin-map-layers-e4e0f3ca-b87b-4fe2-aa2c-57a03c83157d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Test Migration to Vitest",
```

Update the comment, the testing was good and we've deemed the migration viable

---

the whole purpose of these change files is to communicate to the user changes to the api, they are supposed to be user facing. changing the internal test framework doesnt need to be shared explicitly via users in the changelog

```suggestion
  "comment": "",
```